library/Notifications/Widget/ItemList/IncidentList.php

```diff
@@ -24,6 +25,9 @@ protected function init(): void
         $this->on(self::ON_ASSEMBLED, function () {
             ObjectsRendererHook::load();
         });
+
+        $this->getAttributes()
+            ->set('data-icinga-detail-url', (string) Url::fromPath('notifications/incident'));
```

Comment:
Same change as for `ContactGroupList`.
Remember to introduce null values to `Links::incident()` as well.